x/oracle/types/keys_test.go

```diff
@@ -0,0 +1,108 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+	"github.com/umee-network/umee/v2/app"
+	"github.com/umee-network/umee/v2/x/oracle/types"
+)
+
+const (
+	denomLunaIBC = "ibc/0EF15DF2F02480ADE0BB6E85D9EBB5DAEA2836D3860E9F97F9AADE4F57A31AA0"
+	denomAtomIBC = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
```

Comment:
Rename suggestion : 

```suggestion
	ibcDenomLuna = "ibc/0EF15DF2F02480ADE0BB6E85D9EBB5DAEA2836D3860E9F97F9AADE4F57A31AA0"
	ibcDenomAtom = "ibc/27394FB092D2ECCD56123C74F36E4C1F926001CEADA9CA97EA622B25F41E5EB2"
```

+ can we move these to a central location and use them in both files? maybe to the top of `test_utils.go`